tests/client/object_security.c

```diff
@@ -48,7 +48,7 @@
 
 #define LWM2M_SECURITY_URI_ID                 0
 #define LWM2M_SECURITY_BOOTSTRAP_ID           1
-#define LWM2M_SECURITY_SECURITY_ID            2
+#define LWM2M_SECURITY_MODE_ID            2
```

Comment:
These constants are actually defined in multiple places. My bad...
This change should be made in liblwm2m.h. client, lightclient and bootstrap server codes would need to be updated.